pset-iterators.js

```diff
@@ -43,6 +43,19 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents
```

Comment:
@jacobkelman nice approach - you're chaining the methods you ensure that you are getting the all the Whigs and then using `.map()` to get the first names.

An alternative approach would be to use `.find()` to get the first president that matched the criteria and just extract the name afterwards